includes/bootstrap/SWSFeatureContext.php

```diff
@@ -300,6 +300,26 @@ public function iClickOnTheElementWithCSSSelector($cssSelector) {
   }
 
   /**
+   * Press the element with the provided CSS Selector
+   *
+   * @When /^I press the element with css selector "([^"]*)"$/
+   */
+  public function iPressTheElementWithCSSSelector($cssSelector) {
+    $session = $this->getSession();
+    $xpath = $session->getSelectorsHandler()->selectorToXpath('css', $cssSelector);
+    $element = $session->getPage()->find(
+        'xpath',
+        $xpath
+    );
+
+    if (null === $element) {
+        throw new \InvalidArgumentException(sprintf('Could not evaluate CSS Selector: "%s"', $cssSelector));
+    }
+
+    $element->press();
```

Comment:
Is there a difference between $element->press and $element->click? I am assuming press is for button elements only. This step definition then would be for buttons that do not have specific enough labels?